integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -0,0 +1,94 @@
+const _ = require('lodash/fp')
+const fetch = require('node-fetch')
+const pRetry = require('p-retry')
+const { withWorkspace } = require('../utils/integration-helpers')
+const { click, clickable, delay, dismissNotifications, findElement, findText, fillIn, input, select, signIntoTerra, waitForNoSpinners, findInGrid, navChild } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const snapshotName = 'testsnapshot'
+const findWorkflowButton = clickable({ textContains: 'Find a Workflow' })
+
+const withDataRepoCheck = test => async options => {
+  const { dataRepoUrl } = options
+  const res = await fetch(`${dataRepoUrl}/status`)
+  if (res.status === 200) {
+    await test(options)
+  } else {
+    console.error('Skipping data repo snapshot test, API appears to be down')
+  }
+}
+
+const testRunWorkflowOnSnapshotFn = _.flow(
+  withWorkspace,
+  withUserToken,
+  withDataRepoCheck
+)(async ({ dataRepoUrl, page, testUrl, snapshotColumnName, snapshotId, snapshotTableName, token, workflowName, workspaceName }) => {
+  if (!snapshotId) {
+    return
+  }
+  // IMPORT SNAPSHOT
+  await page.goto(`${testUrl}/#import-data?url=${dataRepoUrl}&snapshotId=${snapshotId}&snapshotName=${snapshotName}&format=snapshot`)
+  await signIntoTerra(page, token)
+  await dismissNotifications(page)
+  await click(page, clickable({ textContains: 'Start with an existing workspace' }))
+  await select(page, 'Select a workspace', workspaceName)
+  await click(page, clickable({ text: 'Import' }))
+
+  // ADD WORKFLOW
+  await click(page, navChild('workflows'))
+  await findElement(page, findWorkflowButton)
+  await waitForNoSpinners(page)
+  await click(page, findWorkflowButton)
+  await click(page, clickable({ textContains: workflowName }))
+  await waitForNoSpinners(page)
+  await click(page, clickable({ text: 'Add to Workspace' }))
+
+  // START WORKFLOW ON SNAPSHOT
+  // note that this automatically brings in the highest numbered config, which isn't what happens when going through the method repo in FC
+  await waitForNoSpinners(page)
+  await select(page, 'Entity type selector', snapshotName)
+  await select(page, 'Snapshot table selector', snapshotTableName)
+
+  await click(page, clickable({ textContains: 'Inputs' }))
+  await (await findElement(page, input({ labelContains: 'echo_strings.echo_to_file.input1' }))).click({ clickCount: 3 })
```

Comment:
Added a comment - it's a triple click. The input box already contains a default string. Without the triple click, we simply append. Triple clicking highlights the string so that we can replace it